client/blocks/hello-vote-notice/index.jsx

```diff
@@ -70,6 +72,10 @@ export default connect(
 				isSiteSection( state ) &&
 				'settings' !== getSectionName( state ) &&
 				canCurrentUser( state, selectedSiteId, 'manage_options' ) &&
+				(
+					! userStartDate ||
+					moment( userStartDate ).isBefore( moment().subtract( 1, 'weeks' ) )
+				) &&
```

Comment:
yes, this looks a bit out-of-place to me too...

`import { isNewUser } from 'state/ui/guided-tours/contexts';`